src/OmniSharp.Roslyn.CSharp/Services/Navigation/GotoDefinitionServiceV2.cs

```diff
@@ -83,6 +99,11 @@ public async Task<GotoDefinitionResponse> Handle(GotoDefinitionRequest request)
 
                 return new GotoDefinitionResponse();
             }
+
+            bool IsDecompiledSource(SyntaxTree? syntaxTree)
```

Comment:
Since I imagine this would also work for the normal metadata source, perhaps a more general name.

```suggestion
            bool IsMetaDataSource(SyntaxTree? syntaxTree)
```

Reply:
Changed.